design/ul/concepts.md

```diff
@@ -0,0 +1,17 @@
+# Releasy Ubiquitous Language
+
+Feature
+
+Branch / Feature branch
```

Comment:
We didn't decided yet, that `Feature branch` is an element of our domain.
Do we let it and we will remove it later or you remove it and we'll added it later if needed?